modules/virt-building-real-time-container-disk-image.adoc

```diff
@@ -3,163 +3,98 @@
-
-To build a container disk image, you must create an image builder virtual machine (VM). The _image builder VM_ is a {op-system-base} 8 VM that can be used to build custom {op-system-base} images.
-
-.Prerequisites
-* The image builder VM must run {op-system-base} 8.7 and must have a minimum of 2 CPU cores, 4 GiB RAM, and 20 GB of free space in the `/var` directory.
-* You have installed the image builder tool and its CLI (`composer-cli`) on the VM.
-
-* You have installed the `virt-customize` tool by using the following command:
-+
-[source,terminal]
-----
-# dnf install libguestfs-tools
-----
-* You have installed the Podman CLI tool (`podman`).
+You can create a custom {op-system-base-full} 8 Operating system image in `QCOW2` format and use it to create a container disk image. You can store the container disk image in a registry that is accessible from your cluster and specify the image location in the `spec.param.vmUnderTestContainerDiskImage` attribute of the realtime checkup config map.
 
 .Procedure
 
-. Verify that you can build a {op-system-base} 8.7 image:
-+
-[source,terminal]
-----
-# composer-cli distros list
-----
-+
-[NOTE]
-====
-To run the `composer-cli` commands as non-root, add your user to the `weldr` or `root` groups:
-
-[source,terminal]
-----
-# usermod -a -G weldr user
-----
-[source,terminal]
-----
-$ newgrp weldr
-----
-====
-
-. Enter the following command to create an image blueprint file in TOML format that contains the packages to be installed, kernel customizations, and the services to be disabled during boot time:
-+
-[source,terminal]
-----
-$ cat << EOF > real-time-vm.toml
-name = "realtime_image"
-description = "Image to use with the real-time checkup"
-version = "0.0.1"
-distro = "rhel-87"
-
-[[customizations.user]]
+. Run a {op-system-base-full} 8 Operating system image create a new root user with the following user and password:
 name = "root"
 password = "redhat"
 
-[[packages]]
-name = "real-time"
-
-[[packages]]
-name = "real-time-tools"
-
-[[packages]]
-name = "driverctl"
-
-[[packages]]
-name = "tuned-profiles-cpu-partitioning"
-
-[customizations.kernel]
-append = "default_hugepagesz=1GB hugepagesz=1G hugepages=1"
-
-[customizations.services]
-disabled = ["NetworkManager-wait-online", "sshd"]
-EOF
-----
-
-. Push the blueprint file to the image builder tool by running the following command:
-+
-[source,terminal]
-----
-# composer-cli blueprints push realtime-vm.toml
-----
-
-. Generate the system image by specifying the blueprint name and output file format. The Universally Unique Identifier (UUID) of the image is displayed when you start the compose process.
+. login to this root user and install the following packages:
 +
 [source,terminal]
 ----
-# composer-cli compose start realtime_image qcow2
+# dnf install kernel-rt tuned tuned-profiles-realtime tuned-profiles-nfv-guest rt-tests cloud-init
```

Comment:
In the guest OS configuration instructions, there is a step for enabling the real-time and NFV repos.
Please consider explicitly mentioning it or making the step more general by not showing the explicit command.

Reply:
ACK. DONE